pkg/lint/rules/template.go

```diff
@@ -50,7 +50,7 @@ func Templates(linter *support.Linter, values map[string]interface{}, namespace
 	path := "templates/"
 	templatesPath := filepath.Join(linter.ChartDir, path)
 
-	templatesDirExist := linter.RunLinterRule(support.WarningSev, path, validateTemplatesDir(templatesPath))
+	templatesDirExist := linter.RunLinterRule(support.ErrorSev, path, validateTemplatesDir(templatesPath))
```

Comment:
I don't think the change should be here. An absent `templates` directory should only be a warning as it does not affect a chart being deployed. See below for removing the check on template directory exists.